package.json

```diff
@@ -49,8 +49,8 @@
   },
   "dependencies": {
     "algoliasearch": "^3.8.0",
-    "algoliasearch-helper": "^2.4.0",
-    "classnames": "^2.1.5",
+    "algoliasearch-helper": "^2.5.0",
+    "classnames": "^2.1.3",
```

Comment:
Well the new version of the helper is a must so I need to update this !#(R$&$ shrinkwrap ;P